test/python/WMCore_t/WMException_t.py

```diff
@@ -21,13 +23,27 @@ def setUp(self):
         setup log file output.
         """
         logging.basicConfig(level=logging.NOTSET,
-            format='%(asctime)s %(name)-12s %(levelname)-8s %(message)s',
-            datefmt='%m-%d %H:%M',
-            filename='%s.log' % __file__,
-            filemode='w')
+                            format='%(asctime)s %(name)-12s %(levelname)-8s %(message)s',
+                            datefmt='%m-%d %H:%M',
+                            filename='%s.log' % __file__,
+                            filemode='w')
 
         self.logger = logging.getLogger('WMExceptionTest')
 
+        self.test_data = {
+            'key-ascii': 'value-1',       # unicode (ascii): unicode (ascii)
+            'key-ascii-b': '√@ʟυℯ-1',     # unicode (ascii): unicode (non-ascii)
+            'key-ascii-c': u'√@ʟυℯ-1',    # unicode (ascii): unicode (non-ascii)
+            'key-ascii-d': bytes('√@ʟυℯ-1', 'utf-8'),    # unicode (ascii): bytes (of non-ascii)
+            'ḱℯƴ-unicode-a': 'ṽ@łυ℮-2',   # unicode (non-ascii): unicode (non-ascii)
+            u'ḱℯƴ-unicode-b': 'ṽ@łυ℮-2',  # unicode (non-ascii): unicode (non-ascii)
+            bytes('ḱℯƴ-unicode-c', 'utf-8'): 'ṽ@łυ℮-2',  # bytes (of non-ascii): unicode (non-ascii)
```

Comment:
Can you add another test case where either the key or value (or both) is: `'\x95'` :-D

Reply:
Aha, I get your point. '\x95' does not seem to be a valid unicode-encoded sequence of bytes, so you hope to break the decodings with this and force me to add the "ignore" argument. Nice, thanks!